vue-frontend/src/components/mobile-app-development/CTASection2.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <section class="tw-relative tw-bg-black-core/[0.85] tw-py-[50px]">
+    <div
+      class="tw-absolute tw-right-0 tw-top-1 tw-hidden tw-h-[140px] tw-border-l-[20px] tw-border-[#F2709C] md:tw-block lg:tw-top-[2.25rem] xl:tw-h-[185px] 2xl:tw-top-[4.25rem]"
+    ></div>
+    <div
+      class="tw-absolute tw-bottom-[6.25rem] tw-left-0 tw-hidden tw-h-[115px] tw-border-l-[20px] tw-border-[#FF9472] md:tw-block"
+    ></div>
+
+    <div
+      class="tw-container tw-flex tw-flex-col tw-items-center md:tw-flex-row md:tw-flex-row-reverse"
+    >
+      <div
+        class="tw-flex tw-flex-col md:tw-relative md:tw-items-end lg:tw-flex-[55%]"
+      >
+        <p
+          class="tw-text-center tw-font-inter-bold tw-text-[2.625rem] tw-leading-[3.125rem] tw-text-white md:tw-text-right md:tw-text-[3.275rem] md:tw-leading-[3.875rem] xl:tw-text-[5.625rem] xl:tw-leading-[6.125rem]"
+        >
+          <span class="tw-bg-gradient-L tw-bg-clip-text tw-text-transparent"
+            >L</span
+          >et's
+          <span
+            class="tw-mr-[-0.2rem] tw-bg-gradient-W tw-bg-clip-text tw-text-transparent"
+            >W</span
+          >ork
+          <span
+            class="tw-mr-[-0.2rem] tw-bg-gradient-T tw-bg-clip-text tw-text-transparent"
+            >T</span
+          >ogether
+        </p>
+        <p
+          class="tw-mt-6 tw-w-[385px] tw-text-center tw-font-inter-regular tw-text-[1rem] tw-leading-[1.5rem] tw-text-white md:tw-text-right md:tw-text-[1.4rem] md:tw-leading-[2rem] xl:tw-w-full xl:tw-text-[1.75rem] xl:tw-leading-[2.625rem]"
```

Comment:
i think you haven't added opacity to text. and also on desktop text alignment is not looking proper